extensions/groovy-actions/services/src/main/resources/META-INF/cxs/mappings/groovyAction.json

```diff
@@ -0,0 +1,20 @@
+{
+  "dynamic_templates": [
+    {
+      "all": {
+        "match": "*",
+        "match_mapping_type": "string",
+        "mapping": {
+          "type": "text",
+          "analyzer": "folding",
+          "fields": {
+            "keyword": {
+              "type": "keyword",
+              "ignore_above": 256
+            }
+          }
+        }
+      }
+    }
+  ]
```

Comment:
Since this type will not change, I think it would be best to put the full mapping for it.